src/main/java/website/frontrow/board/Player.java

```diff
@@ -158,7 +178,8 @@ public void loseLife()
     public void addLife()
     {
     	lives++;
-    	addToLog("[PLAYER]\t Player earned a life, total lives is now: " + lives);
+        observers.forEach((o) -> o.livesChanged(this));
```

Comment:
Indenting, line 180 has a tab, line 181 only has spaces.
There's a visible difference, because of this.